exercises/proverb/example.js

```diff
@@ -0,0 +1,35 @@
+module.exports = function(){
+  const last = arguments[arguments.length - 1];
```

Comment:
We'll need to make this a `var` for the sake of consistency, and because this track emphasizes ES5.